tests/serializer/json_serializer.py

```diff
@@ -149,6 +149,7 @@ def testReadAndWriteSerializedEventObject(self):
     expected_event_object.empty_string = ''
     expected_event_object.zero_integer = 0
     expected_event_object.integer = 34
+    expected_event_object.float = -122.082203542683
```

Comment:
maybe rename event_object to event as well

Reply:
Done